lib/blacklight/search_builder.rb

```diff
@@ -21,6 +21,13 @@ def with blacklight_params = {}
     end
 
     ##
+    # Update the :q (query) parameter
+    def where conditions
```

Comment:
@jcoyne I wonder if we actually need `where` vs adding a `append_params` or similar method.